src/server/src/routes/student/schemas.ts

```diff
@@ -24,12 +24,10 @@ export const studentSearchSchema = Joi.object({
     lastName: Joi.string().allow('').optional(),
     email: Joi.string().allow('').optional(),
     sid: Joi.string().allow('').optional(),
-    departmentIds: Joi.array()
-        .items(Joi.number())
-        .optional(),
-    classStandings: Joi.array()
+    departmentIds: Joi.array().allow('').items(Joi.number()).required(),
```

Comment:
Question: Why is departmentIds allow to receive an empty string?

Reply:
If I do not have "allow('')", I cannot have an empty array as an input on postman.